### PR TITLE
[Docs] softmax_cross_entropy_with_integer_label

### PR DESCRIPTION
**Pull request by lkhphuc**
Document for the new loss function is missing from Readthedocs.

### CODE REVIEW

**Review by mkunesch — APPROVED** [review]
Thanks a lot for spotting this! (and sorry for the slow review)
